src/conn/conn_cache_pool.c

```diff
@@ -156,6 +161,9 @@ __wt_cache_pool_config(WT_SESSION_IMPL *session, const char **cfg)
 		    session, cfg, "shared_cache.chunk", &cval));
 		WT_ASSERT(session, cval.val != 0);
 		chunk = (uint64_t)cval.val;
+		WT_ERR(__wt_config_gets(
+		    session, cfg, "shared_cache.quota", &cval));
+		quota = (uint64_t)cval.val;
 	}
```

Comment:
Three is about as many options as I'd like to repeat here -- is it reasonable to create an inlined function that handles both cases and call it three times?